orienteer-core/src/main/java/org/orienteer/core/web/OrienteerBasePage.html

```diff
@@ -12,6 +12,9 @@
 		<span class="navbar-toggler-icon"></span>
 	</button>
 
+	<nav class="navbar navbar-expand-lg" wicket:id="topMenu">
```

Comment:
Is menu looks good from mobile? Menu should be just collapsed.

Reply:
It is bootstrap menu. I think mobile devices supported.